app/services/html/parser.rb

```diff
@@ -55,6 +62,13 @@ def prefix_all_images(width = 880)
       self
     end
 
+    def image_height_width(img)
+      src = img.attr("src")
+      return unless src
+
+      FastImage.size(src, timeout: TIMEOUT) # outputs as [width, height]
```

Comment:
Wouldn't it make more sense to call this method `image_width_height` then which may also make the explanatory comment unnecessary?